bench/workgen/workgen.cxx

```diff
@@ -428,21 +428,15 @@ ContextInternal::ContextInternal()
 
 ContextInternal::~ContextInternal()
 {
-    if (_table_runtime != nullptr)
-        delete _table_runtime;
+    if (_table_runtime.size() != 0)
+        _table_runtime.resize(0);
```

Comment:
You don't need to resize to destroy the allocated memory. Actually resizing doesn't delete the vector's internally allocated memory. You can remove the destructor and just use an implicit one at this point, the destruction of ContextInternal will take care of cleanup.